tamr_client/datasets/__init__.py

```diff
@@ -1,2 +1,5 @@
 # This __init__.py file is necessary for `mypy --package`
```

Comment:
we can remove this comment since it was just here to justify the presence of an empty `__init__.py` file, but it's not empty anymore.